### DISCUSSION TITLE
500: INTERNAL_SERVER_ERROR

### DISCUSSION THREAD

**Discussion by sudhamrit** [vercel/community#2849] [Bug Report]
### Page to Investigate

https://sudhamrit-fp4anoacr-sudhamrit.vercel.app/

### Steps to Reproduce

The app was running properly till yesterday, but now when we visit the domain, we are getting the following error

500: INTERNAL_SERVER_ERROR
Code: INTERNAL_FUNCTION_INVOCATION_FAILED
ID: bom1::vwfx6-1686684404880-fca4202b21c4

**Comment by amyegan** [comment]
There is an open incident related to serverless functions: https://www.vercel-status.com/incidents/cqkdqrzqkspm

Can you switch to a region other than `iad1` and redeploy? Please let me know if that solves the problem for you 🙏

**Reply by sudhamrit** [reply]
Thnanks @amyegan after the region change it's working !!

**Reply by xardass** [reply]
Thanks! This fixed it for us as well. Changed to Paris

**Reply by eurunuela** [reply]
It hasn't fixed it for me...
